text_collector_examples/ipmitool

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/awk -nf
+
+#
+# Converts output of `ipmitool sensor` to prometheus format.
+#
+# With GNU awk:
+#   ipmitool sensor | ./ipmitool > ipmitool.prom
+#
+# With BSD awk:
+#   ipmitool sensor | awk -f ./ipmitool > ipmitool.prom
+#
+
+function export(values, name) {
+	if (values["metric_count"] < 1) {
+		return
+	}
+	delete values["metric_count"]
+
+	printf("# HELP %s%s %s sensor reading from ipmitool\n", namespace, name, friendly[name]);
+
+	if (name != "status") {
```

Comment:
Why is this not printing the type?  The status is a gauge.  From a Prometheus perspective, basically anything that isn't a monotonic increasing counter is considered a gauge.

I would remove the `if` so that you always get a `TYPE` line.